### DISCUSSION TITLE
Entry Condition: be able to filter on empty or non-empty Date/Time field

### DISCUSSION THREAD

**Discussion by romainpoirier** [craftcms/cms#12066] [Ideas] [answered]
It would be useful to be able to use Entry Condition to Only show fields when editing entries that match the following rules: a Date/Time field is empty or non-empty.

Currently, we can only define a date using `From` & `To`, so the show/hide condition is based on the specific content, not if it's empty or not empty.

**Comment by brandonkelly** [comment, marked as answer]
This was made possible in Craft 4.3 🎉 (#11070)
